mitodl/open-discussions
# Updated post/comment APIs to enable ignoring future reports

#### What are the relevant tickets?
Fixes #427 

#### What's this PR do?
Adds functionality to comment/post APIs to ignore future reports on the items

#### How should this be manually tested?
- Report a comment and a post
- Verify the reports show up on `/api/v0/channels/:channel_name/reports/`
- Go to `/api/v0/posts/:post_id` and `/api/v0/comments/:comment_id` and patch with `{"ignore_reports": true}`
- Verify the reports no longer show up on `/api/v0/channels/:channel_name/reports/`
- Report the comment and post again under a **different user**
- Verify the reports still don't show up on `/api/v0/channels/:channel_name/reports/`
